ex13.py

```diff
@@ -0,0 +1,11 @@
+from sys import argv
```

Comment:
1. Requests. The most famous http library.
2. Scrapy. This is used for webscraping.
3. wxPython. A gui toolkit for python.
4. Pillow. A PIL (Python Imaging Library) basically used when working with images.
5. SQLAlchemy. A database library.
6. NumPy. It provides some advance math functionalities to python.
7. SciPy. It is a library of algorithms and mathematical tools for python.
8.Twisted. This is used for the API purpose.